web/src/model.rs

```diff
@@ -0,0 +1,188 @@
+        false
+    }
+
+    fn view(&self) -> Html {
+        html! {
+            <div class="container main spacing-v-xl">
+                <div class="form-grid">
+                    <div class="form-grid__row form-grid__row--small">
+                        <legend class="form-legend">
+                            <span class="form-legend__title">{"LOCALAUTH0"}</span>
+                            <span class="form-legend__addon">
+                                <img
+                                    src="assets/static/media/localauth0.png"
+                                    width="80"
+                                    height="80"
+                                    alt="Localauth0 logo"
+                                />
+                            </span>
+                        </legend>
+                    </div>
+
+                    <div class="form-grid__row form-grid__row--small">
+                        <div class="form-item">
+                            <label class="form-label" for="audience">{ "Audience" }</label>
+                            <div class="form-item__wrapper">
+                                <div class="form-field">
+                                    <label class="form-field__wrapper">
+                                        <input id="form-item-name" class="form-field__text" name="audience" type="text" placeholder="audience" ref={self.audience_input_ref.clone()}/>
+                                    </label>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+
+                    {{self.permission_input_view()}}
+
+                    { for self.permissions.iter().map(|permission| self.view_entry(permission.to_string())) }
+
+                    <div class="form-grid__row form-grid__row--small">
+                        <div class="form-item">
+                            <label class="form-label" for="label-and-textarea">{ "Token" }</label>
+                            <div class="form-item__wrapper">
+                                <div class="form-field">
+                                    <textarea
+                                        id="label-and-textarea"
+                                        class="form-field__textarea token-area"
+                                        readonly=true>
+                                        {self.token.clone().map(|jwt| jwt.access_token).unwrap_or("No token".to_string())}
+                                    </textarea>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+
+                    <div class="form-grid__row form-grid__row--small">
+                        <button class="button button--primary button--huge" onclick=self.link.callback(|_| Msg::SetPermissions)>{"Generate token"}</button>
+                    </div>
```

Comment:
Anche qui manca una colonna e il `button-row` (so che l'avete levato per mettere il bottone centrale a tutta larghezza, ma provate così:

```suggestion
                    <div class="form-grid__row form-grid__row--small">
                        <div class="form-grid__row__column">
                            <div class="button-row button-row--center">
                                <button />
                            </div>
                        </div>
                    </div>
```

Nel `main.scss` aggiungete pure una classe `.button-row--center`, poi la faremo anche noi 👍 

```
.button-row--center {
  justify-content: center;
}